src/override/malloc.cc

```diff
@@ -135,7 +135,12 @@ extern "C"
       return nullptr;
     }
 
-    uint8_t sc = size_to_sizeclass((std::max)(size, alignment));
+    size = (std::max)(size, alignment);
+    if (size >= SUPERSLAB_SIZE)
+    {
```

Comment:
Why is this necessary?  Doesn't the subsequent test sc>=NUM_SIZECLASSES cover this?  Is this an optimisation, or am I missing something in the logic.

Reply:
It wasn't. We just need to set size as the max of size and alignment for the following allocation